code/controllers/AssetAdmin.php

```diff
@@ -25,28 +16,6 @@ class AssetAdmin extends LeftAndMain implements PermissionProvider{
 	private static $page_length = 15;
 	
 	/**
-	 * @config
-	 * @see Upload->allowedMaxFileSize
-	 * @var int
-	 */
-	private static $allowed_max_file_size;
```

Comment:
That removes the config hint which we might start parsing into something useful soon